### PR TITLE
Update content-version-cleanup.md

### PR DESCRIPTION
**Pull request by Jette94**
Added relevant information from https://docs.umbraco.com/umbraco-cms/reference/configuration/contentsettings#enablecleanup

## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 14/umbraco-cms/fundamentals/data/content-version-cleanup.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Additionally, it is possible to keep the feature enabled but mark specific versi
 
 It is worth noting that whilst we delete rows, we do not shrink database files or rebuild indexes. For upgraded sites with a lot of history you may wish to perform these tasks. If they are not part of your regular database maintenance plan already.
 
+If set to `true`, a scheduled job will delete historic content versions every hour, while setting it to `false` will never delete any content versions regardless of settings. For more information on this, see [EnableCleanup](../../reference/configuration/contentsettings.md#enablecleanup).
```
Comment (sofietoft) [inline 14/umbraco-cms/fundamentals/data/content-version-cleanup.md:49]: I've rewritten this a bit!
And what do you think about putting it further up in the article, like right above the "How it works" heading?

```suggestion
A scheduled job will delete content versions every hour when the Content Version Cleanup Policy is enabled. When not enabled, no content versions will be deleted regardless of settings. For more information on this, see [EnableCleanup](../../reference/configuration/contentsettings.md#enablecleanup).
```

**Comment by sofietoft** [comment]
@Jette94 - Did you see my comment about moving up the sentence a bit? Like putting it right above the "How it works" heading?

File: 14/umbraco-cms/fundamentals/data/content-version-cleanup.md [hunk 2]
```diff
@@ -46,6 +46,8 @@ Additionally, it is possible to keep the feature enabled but mark specific versi
 
 It is worth noting that whilst we delete rows, we do not shrink database files or rebuild indexes. For upgraded sites with a lot of history you may wish to perform these tasks. If they are not part of your regular database maintenance plan already.
 
+A scheduled job will delete content versions every hour when the Content Version Cleanup Policy is enabled. When not enabled, no content versions will be deleted regardless of settings. For more information on this, see [EnableCleanup](../../reference/configuration/contentsettings.md#enablecleanup).
```
Comment (sofietoft) [inline 14/umbraco-cms/fundamentals/data/content-version-cleanup.md:49]: As per Bjarkes note about not having the same information twice 🙌 

```suggestion
A scheduled job will delete content versions regularly when the Content Version Cleanup Policy is enabled. For more information on how often content is deleted, read the [ContentSettings article](../../reference/configuration/contentsettings.md#enablecleanup).
```

**Comment by Jette94** [comment]
Like this, @sofietoft ? :D

**Comment by sofietoft** [comment]
Looks great @Jette94 ! 😄 

I'll get it merged!

**Pull request merged**